README.md

```diff
@@ -0,0 +1,72 @@
+# Hardly
+
+Code for (Celery) worker used in our "Stream service".
+The service is similar to [Packit service](https://github.com/packit/packit-service),
+but follows [this workflow](https://github.com/packit/research/tree/main/split-the-stream#what-does-the-source-git-workflow-mean).
+
+It reuses [packit-service's worker](https://github.com/packit/packit-service/tree/main/packit_service/worker)
+code and [implements new jobs as new handlers](https://github.com/packit/research/tree/main/split-the-stream#2-separate-workers).
+
+## Running it locally
+
+[Similar to packit-service](https://github.com/packit/packit-service/blob/main/CONTRIBUTING.md#running-packit-service-locally)
+we have a [docker-compose.yml](docker-compose.yml) for fast prototyping.
+You might need to tweak it a bit (especially the volume mounts there) before running it,
+because it expects that you have:
+
+- secrets/stream/dev/ populated (linked) with secrets mostly taken from our internal repo.
+  "Mostly", because you should use your credential where possible.
+- ../ogr/, ../packit/ and ../packit-service/ dirs with the respected repos cloned.
+  Those are mounted into the container so you don't have to rebuild the image each time you change anything in them.
+
+Follow the [packit-service's guide](https://github.com/packit/packit-service/blob/main/CONTRIBUTING.md#running-packit-service-locally)
+for the other settings. Once you have it running (and see no errors), you can test (uses [HTTPie](https://httpie.io)) the webhook with:
+
+```
+cat tests/data/webhooks/gitlab/mr_event.json | http --verify=no https://dev.packit.dev:8443/api/webhooks/gitlab
+```
+
+## Deployment
+
+To deploy the service into Openshift cluster,
+clone the [deployment repo](https://github.com/packit/deployment) and:
+
+1. [create the variable files](https://github.com/packit/deployment/tree/main/vars) in [vars/stream/](https://github.com/packit/deployment/tree/main/vars/stream)
+2. link secrets into [secrets](https://github.com/packit/deployment/tree/main/secrets) / stream/{prod|stg}
+3. `SERVICE=stream DEPLOYMENT={deployment} make deploy`
+
+## Image
+
+[The image](files/Containerfile) is currently based on the
+[packit-worker image](https://github.com/packit/packit-service/blob/main/files/docker/Dockerfile.worker)
+but that might change in the future to decouple those.
+
+For running locally with docker-compose, build it with `docker-compose build`.
```

Comment:
`podman-compose build` worked for me, and although I didn't set up the repo correctly, `podman-compose up` also seemed to work.

Reply:
Thanks for trying, yes `podman-compose build` works for me as well, but I'm not able to make it work with `podman-compose up`.
I removed the `user: "123123"` and changed some container names (suggested in https://github.com/containers/podman-compose/issues/243#issuecomment-802691015) but still no luck.
So I'll leave the instruction docker-compose specific.